virtualizarr/tests/test_kerchunk.py

```diff
@@ -232,3 +232,26 @@ def test_FileType():
     assert "zarr" == FileType("zarr").name
     with pytest.raises(ValueError):
         FileType(None)
+
+
+@pytest.mark.parametrize(
+    "format",
+    [
+        "json",
+        pytest.param(
+            "parquet", marks=pytest.mark.xfail(reason="parquet reading not finished")
+        ),
+    ],
+)
+def test_kerchunk_to_virtual_dataset(netcdf4_file, tmpdir, format):
+    vds = open_virtual_dataset(netcdf4_file, indexes={})
+
+    # QUESTION: should these live in a fixture? ex. kerchunk_ref_fpath_json, kerchunk_ref_fpath_parquet
```

Comment:
eh you kind of want the original vds as well as the kerchunk refs so I think it is fine as is.